app/models/topic_embed.rb

```diff
@@ -132,7 +132,7 @@ def self.find_remote(url)
 
     response = FetchResponse.new
     begin
-      html = uri.read(allow_redirections: :safe)
+      html = uri.read
```

Comment:
Sorry just saw this, is `allow_redirections: :safe` the new default? I didn't dig too deep but just want to make sure we are not potentially regressing on a security problem.

Reply:
Yes that’s the default since Ruby 2.4 actually :slightly_smiling_face:

Reply:
:+1: Thank you for clarifying.